configs/raspberrypi3_wpe_rdk_defconfig

```diff
@@ -18,12 +18,11 @@ BR2_ROOTFS_POST_BUILD_SCRIPT="board/raspberrypi3/post-build.sh"
 BR2_ROOTFS_POST_IMAGE_SCRIPT="board/raspberrypi3/post-image.sh"
 BR2_ROOTFS_POST_SCRIPT_ARGS="--add-pi3-miniuart-bt-overlay"
 BR2_LINUX_KERNEL=y
-BR2_LINUX_KERNEL_CUSTOM_GIT=y
-BR2_LINUX_KERNEL_CUSTOM_REPO_URL="https://github.com/raspberrypi/linux.git"
-BR2_LINUX_KERNEL_CUSTOM_REPO_VERSION="9126e25b0934bd7bd843763310ea4b34c6e139d0"
+BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,raspberrypi-kernel_1.20180313-1)/raspberrypi-kernel_1.20180313-1.tar.gz"
```

Comment:
I have checked Buildroot stable (2018.08.2), LTS (2018.02.7) releases, plus the current `master` from Git, and this version is not used at all upstream Buildroot. Could you please elaborate where is this coming from, and why the same version as used by upstream Buildroot is not good?

Reply:
I would go with (1.) and using the latest Buildroot release. It would be just perfect if you could update the PR to use that, thanks a ton for the clarification!

Reply:
Updated!

Reply:
Thanks! This PR looks now perfect; let's merge it :muscle: